src/nvme/mi.c

```diff
@@ -18,6 +17,12 @@
 #include "mi.h"
 #include "private.h"
 
+/* 01b: host to controller */
+#define IS_DIRECTION_WRITE(op)  (((op)&0x3) == 0x1)
```

Comment:
Use static inline instead macros for this kind of stuff.

Reply:
Removed.